$NetBSD: patch-tools_Makefile,v 1.5 2016/02/20 01:42:24 khorben Exp $

Do not build gdeasm for now (as it needs an additional library)

--- tools/Makefile.orig	2016-02-20 01:28:33.000000000 +0000
+++ tools/Makefile
@@ -1,5 +1,5 @@
 SUBDIRS	= backend debug models
-TARGETS	= $(OBJDIR)console$(EXEEXT) $(OBJDIR)debugger$(EXEEXT) $(OBJDIR)gdeasm$(EXEEXT) $(OBJDIR)sequel$(EXEEXT) $(OBJDIR)simulator$(EXEEXT)
+TARGETS	= $(OBJDIR)console$(EXEEXT) $(OBJDIR)debugger$(EXEEXT) $(OBJDIR)sequel$(EXEEXT) $(OBJDIR)simulator$(EXEEXT)
 OBJDIR	=
 PREFIX	= /usr/local
 DESTDIR	=
@@ -115,8 +115,6 @@ install: $(TARGETS)
 	$(MKDIR) $(DESTDIR)$(BINDIR)
 	$(INSTALL) -m 0755 $(OBJDIR)debugger$(EXEEXT) $(DESTDIR)$(BINDIR)/debugger$(EXEEXT)
 	$(MKDIR) $(DESTDIR)$(BINDIR)
-	$(INSTALL) -m 0755 $(OBJDIR)gdeasm$(EXEEXT) $(DESTDIR)$(BINDIR)/gdeasm$(EXEEXT)
-	$(MKDIR) $(DESTDIR)$(BINDIR)
 	$(INSTALL) -m 0755 $(OBJDIR)sequel$(EXEEXT) $(DESTDIR)$(BINDIR)/sequel$(EXEEXT)
 	$(MKDIR) $(DESTDIR)$(BINDIR)
 	$(INSTALL) -m 0755 $(OBJDIR)simulator$(EXEEXT) $(DESTDIR)$(BINDIR)/simulator$(EXEEXT)
@@ -128,7 +126,6 @@ uninstall:
 		else $(MAKE) uninstall; fi) || exit; done
 	$(RM) -- $(DESTDIR)$(BINDIR)/console$(EXEEXT)
 	$(RM) -- $(DESTDIR)$(BINDIR)/debugger$(EXEEXT)
-	$(RM) -- $(DESTDIR)$(BINDIR)/gdeasm$(EXEEXT)
 	$(RM) -- $(DESTDIR)$(BINDIR)/sequel$(EXEEXT)
 	$(RM) -- $(DESTDIR)$(BINDIR)/simulator$(EXEEXT)
 
